internal/impl/devbox.go

```diff
@@ -1005,6 +1013,27 @@ func (d *Devbox) findPackageByName(name string) (string, error) {
 	return lo.Keys(results)[0], nil
 }
 
+func (d *Devbox) checkOldEnvrc() error {
+	// check if user has an old version of envrc
+	if fileutil.Exists(".envrc") && os.Getenv("DEVBOX_NO_ENVRC_UPDATE") != "1" {
```

Comment:
Just realized you are using `".envrc"` directly. Instead you should use `filepath.Join(d.ProjecDir(), ".envrc")` in case devbox is called from another dir.

Reply:
@mohsenari this comment applies to all instances on .envrc

This currently breaks if we call it from a directory that contains `.envrc` but the target  project does not (e.g doing anything `global` from a directory with .envrc)